docs/testing/unit-tests.md

```diff
@@ -1,36 +1,42 @@
 # Unit Tests
+
 This guide is intended as a quick reference of common tools and conventions we use for Calypso unit testing. You'll be able to glean a great deal from browsing through Calypso's existing tests, but we'll try to keep this page up to date with the most recent standards.
 
 ## Table of contents
-* [Why unit test?](#why-unit-test)
-* [Running tests](#running-tests)
-* [Writing tests](#writing-tests)
-    * [Folder structure](#folder-structure)
-    * [Importing tests](#importing-tests)
-    * [Describing tests](#describing-tests)
-    * [Snapshot testing](#snapshot-testing)
-    * [Setup and Teardown methods](#setup-and-teardown-methods)
-    * [Mocking dependencies](#mocking-dependencies)
-    * [Testing globals](#testing-globals)
-    * [Testing legacy code](#testing-legacy-code)
-* [Further reading](#further-reading)
+
+- [Unit Tests](#unit-tests)
```

Comment:
Wanted to note that this change felt a bit odd the first glance, but it makes a lot of sense after previewing the parsed document 👌